dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -292,6 +309,7 @@ public class MicroProfileActions {
     public static final FeatureSet MP50 = new FeatureSet(MP50_ID, MP50_FEATURE_SET, EEVersion.EE9);
     public static final FeatureSet MP60 = new FeatureSet(MP60_ID, MP60_FEATURE_SET, EEVersion.EE10);
     public static final FeatureSet MP61 = new FeatureSet(MP61_ID, MP61_FEATURE_SET, EEVersion.EE10);
+    public static final FeatureSet MP70 = new FeatureSet(MP70_ID, MP70_FEATURE_SET, EEVersion.EE10);
 
     //All MicroProfile FeatureSets - must be descending order
     private static final FeatureSet[] ALL_SETS_ARRAY = { MP61, MP60, MP50, MP41, MP40, MP33, MP32, MP30, MP22, MP21, MP20, MP14, MP13, MP12, MP10 };
```

Comment:
`MP70` should be added to `ALL_SETS_ARRAY`, unless there's a reason not to?

Reply:
Be sure to run `io.openliberty.jakartaee10.internal_fat,io.openliberty.jakartaee11.internal_fat,io.openliberty.jakartaee9.internal_fat` in full mode if you change this.

Reply:
We also need to update the `EE*FeatureCompatibilityTest` classes since they currently have an exception for `mpTelemetry-2.0`.

Reply:
Not doing this here, it will be done under #28464